src/support/server_stats.c

```diff
@@ -148,6 +148,8 @@ static const uint32_t nfsv3_optype[NFS_V3_NB_COMMAND] = {
 };
 #endif
 
+#endif
```

Comment:
This doesn't seem correct.  The #ifdef/#endif macros are already balanced in that file, and there's already an endif for the _USE_NFS3 around nfsv3_optype.  It seems this change shouldn't compile?